modules/virt-creating-multiple-storage-pools.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-multiple-storage-pools_{context}"]
+= Creating multiple storage pools
+
+If you have multiple available disks or partitions on your node and want to use all of them for the HPP, use any combination of `storagePools` with or without `pvcTemplates`. 
+
+[NOTE]
+====
+You can use the Local Storage Operator to create provisioned volumes that can used by the HPP.
+====
+
+.Procedure
+
+. Create a YAML file for the HPP CSI driver provisioner custom resource specifying multiple `storagePools`. For example:
++
+[source,terminal]
+----
+$ touch hostpathprovisioner_cr_multi.yaml
+----
+
+. Edit the file. For example:
++
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+    spec:
+      imagePullPolicy: IfNotPresent
+      storagePools: <1>
+      - name: <storage>
+        path: <"/var/mystorage"> <2>
+      - name: <second-disk> <3>
+        pvcTemplate: <4>
+        storageClassName: local-block
+        volumeMode: Block <5>
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 50Gi
+        path: <"/var/myvolumes"> <6>
+ workload:
+   nodeSelector:
+     kubernetes.io/os: linux
+----
+<1> The `storagePools` stanza (an array) specifies multiple storage pools (`storage` and `second-disk`).
+<2> During node start, the `storage` partition is mounted under `/var/storage`. Read/write access required.
+<3> In this example there are two disk volumes on the node.
+<4> Using the `pvctemplate` one disk is partitioned into an operating system and a partition for additional storage. The second disk has been unused since node installation.
+<5> `volumeMode` can be `Block` or `Filesystem`, but must match the provisioned volume format. Default is `Filesystem`. Note that the Local Storage Operator (LSO) writes only in `block` format.
```

Comment:
Maybe mention that in this example the storage class associated with LSO is named local-block.